Makefile

```diff
@@ -101,32 +126,50 @@ generated-code: $(OUTPUT_DIR)/.generated-code update-licenses
 
 SUBDIRS:=pkg test
 $(OUTPUT_DIR)/.generated-code:
-	mkdir -p ${OUTPUT_DIR}
-	rm -rf vendor_any
+	mkdir -p $(OUTPUT_DIR)
 	go mod tidy
-	PATH=$(DEPSGOBIN):$$PATH $(GO_BUILD_FLAGS) go generate ./...
-	PATH=$(DEPSGOBIN):$$PATH gofmt -w $(SUBDIRS)
-	PATH=$(DEPSGOBIN):$$PATH goimports -w $(SUBDIRS)
+	$(GO_BUILD_FLAGS) go generate ./...
+	gofmt -w $(SUBDIRS)
+	goimports -w $(SUBDIRS)
 	touch $@
 
 .PHONY: verify-envoy-protos
 verify-envoy-protos:
 	@echo Verifying validity of generated envoy files...
-	PATH=$(DEPSGOBIN):$$PATH $(GO_BUILD_FLAGS) CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build pkg/api/external/verify.go
-
+	$(GO_BUILD_FLAGS) CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build pkg/api/external/verify.go
 
 #----------------------------------------------------------------------------------
-# Unit Tests
+# Tests
 #----------------------------------------------------------------------------------
 
-# '-skip=multicluster', '-regexScansFilePath' skips any filepath which includes multicluster, which is useful
-# as this code is no longer used
+GINKGO_VERSION ?= 2.5.0 # match our go.mod
```

Comment:
if it matches our go mod can we pull it from the go mod?

Reply:
`go mod graph | grep "github.com/solo-io/solo-projects github.com/onsi/ginkgo" | cut -d '@' -f2 | cut -d 'v' -f2`

Reply:
Happy to. I was lazy and didn't investigate how to do this. Do you have any tips?

Reply:
Yeah so what would be the make syntax? I've been using `go list -m all | grep github.com/onsi/ginkgo/v2 | awk '{print $2}` but I can't set a make variable with that

Reply:
So I thought about this a bit more I have a proposal that is out of scope for this pr as its already too large so ill spin it out into its own issue. Namely we shouldnt be doing dark magic in the make file. We should be checking this with our generate commands that way we can assert all the of the things that change when the go mod changes like we do already with our generate commands in general.